.github/workflows/fpga.yml

```diff
@@ -366,7 +366,9 @@ jobs:
           # Tests that are known to pass on the FPGA
           # TODO(kor): Fix all the tests and run them all here
           sudo ${VARS} "${TEST_BIN}/smoke_test-"* \
-              --skip fips_cmd_test_rt
+              --skip fips_cmd_test_rt \
+              --skip test_rt_wdt_timeout \
+              --skip test_fmc_wdt_timeout
```

Comment:
Do we know why these don't pass on the FPGA?

Reply:
the conditional compilation is not passing the right timeout value.